app/src/main/AndroidManifest.xml

```diff
@@ -23,15 +21,16 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/APP_NAME"
         android:largeHeap="true">
-
+        <activity
+            android:name=".feature.keygear.ui.ReceiptActivity"
```

Comment:
Make sure this has all the stuff from other activities as well, such as configChanges, screenOrientation, etc.